tests/dummy/app/templates/examples.hbs

```diff
@@ -1,3 +1,44 @@
+<h3>Tagging</h3>
+<p>You can provide tags etc.</p>
+
+<div class="row">
+  <div class="col-sm-6">
+      <h4>Ember Data models</h4>
+      <p>Type as much tags as you like.</p>
+      <div class="example-box">
+          <p>Result:</p>
+          <ul>
+            {{#each tag in model.post.tags }}
+              <li>{{tag.name}}</li>
+            {{/each}}
+          </ul>
+        {{#highlight-code lang=".hbs"}}&lt;ul&gt;
+  \{{#each tag in model.post.tags }}
+       &lt;li&gt;\{{tag.name}}&lt;/&gt;
+  \{{/each}}
+&lt;/ul&gt;
+        {{/highlight-code}}
+        <p>Type some things below:</p>
+        {{select-2
+          tags=true
+          content=model.tags
+          value=model.post.tags
+          placeholder="Begin typing tags for this post"
+          optionLabelPath="name"
+          optionIdPath="name"
+        }}
+      </div>
+    {{#highlight-code lang=".hbs"}}\{{select-2
+        tags=true
+        content=model.tags
+        value=model.post.tags
+        placeholder="Begin typing tags for this post"
+        optionLabelPath="name"
+        optionIdPath="name"
+        }}{{/highlight-code}}
+  </div>
+</div>
+
```

Comment:
Please move the examples down on the page, right before `<h3>Miscellaneous</h3>`, once they are done